src/app/pages/datasets/components/dataset-details-card/dataset-details-card.component.html

```diff
@@ -65,6 +65,7 @@ <h3 mat-card-title *ngIf="isZvol">{{ 'Zvol Details' | translate }}</h3>
           matTooltipPosition="above"
           class="value dataset-path"
           [matTooltip]="dataset.name"
```

Comment:
Perhaps a tooltip delay would make sense if there is no way to detect when the tooltip is necessary or not

Reply:
@denysbutenko Can you elaborate a bit? When don't you need the tooltip?

Reply:
I mean cases when the dataset name is not trimmed and fits into cell

Reply:
Initially, my assumption was that we are talking about the dataset tree tooltips and not `dataset-details-card`. Sorry for the confusion. That's not related to this ticket. The `dataset-details-card` looks good and works well 👍🏻